pkg/summary/summarizers.go

```
@@ -218,7 +260,60 @@ func checkStandard(obj data.Object, _ []Condition, summary Summary) Summary {
 	return summary
 }
 
-func checkErrors(_ data.Object, conditions []Condition, summary Summary) Summary {
+func checkGVKErrors(data data.Object, conditions []Condition, summary Summary) (Summary, bool) {
+	if len(conditions) == 0 {
+		return summary, false
+	}
+
+	obj, err := json.Marshal(data)
+	if err != nil {
+		return summary, false
+	}
+
+	gvk, detected, err := gvk.Detect(obj)
+	if err != nil {
+		return summary, false
```

Same here about discarding an error situation